src/components/listItems/listItem.tsx

```diff
@@ -0,0 +1,24 @@
+import './listItem.scss';
+
+interface Props {
+  title: string | number;
+  item?: string | number;
+  last?: boolean;
+}
+
+export const ListItem = ({ title, item, last }: Props) => (
+  <div
+    className="list-item"
+    style={{ borderBlockEnd: last ? 'none' : '2px solid #DFE0EB' }}
```

Comment:
be more verbose 
`itemTitle` -> should only be a string 
`lastItem` 


what does   `item?: string | number; ` mean ??

Reply:
It means that it can either be a string or a number, if it aint any it will default to 'Nan', so the dev using this component that it is either empty or used incorrectly.